framework/src/multiapps/MultiApp.C

```diff
@@ -554,8 +554,10 @@ MultiApp::backup()
 {
   TIME_SECTION(_backup_timer);
 
+  _console << "Backing up MultiApp " << name() << "...";
   for (unsigned int i = 0; i < _my_num_apps; i++)
     _backups[i] = _apps[i]->backup();
+  _console << "done!" << std::endl;
```

Comment:
I dont like these `done!`
Finished etc is a little more professional

Reply:
Will change this to `...finished.`.